MSMapsModules.podspec

```diff
@@ -10,7 +10,7 @@ Library for parsing GeoJSON, extracting the shapes, and adding them to a new Map
   spec.author    = "Microsoft Corporation"
   spec.platform     = :ios, "9.0"
   spec.static_framework = true
-  spec.source       = { :git => 'https://github.com/microsoft/BingMapsNativeModules.git'}
+  spec.source       = { :git => 'file:///Users/admin/Desktop/internProject/BingMapsNativeModules'}
```

Comment:
This is a local path on your machine. This should not be checked in as it brakes the users of your library.